### ISSUE TITLE
Unlock button pull the spring

### ISSUE THREAD

**Issue by bzumik1** [patrickhilker/tedee_hass_integration#34]
Hi, when I click on open button it not only unlock the door but also pull the spring. I have set it up so in tedee app but I would expect it to be independent from HA behavior.

**Comment by patrickhilker** [comment]
You should be able to "unlock" the lock entity. That should do what you want. Please let me know if it doesn't.

**Comment by bzumik1** [comment]
Ok to be honest it looks like issue on my side. I have tested it more carefully and from this screen:

![image](https://github.com/patrickhilker/tedee_hass_integration/assets/32042186/ce0b919d-af6e-4b97-bc96-6f018ec5db54)

it works as expected, but from the device screen:

![image](https://github.com/patrickhilker/tedee_hass_integration/assets/32042186/7cac1801-7304-4631-80c0-c0fbb7d21509)

I have three buttons there and the two bottom ones do the same, unlock the door and pull the spring. I have just discovered that the top one is doing what I want. What should be the difference between those two?
